src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/step_time_field/step_time_field.html

```diff
@@ -2,172 +2,193 @@
+  </button>
 
-    <!-- Index pattern id input -->
-    <div
-      class="kuiVerticalRhythm"
-      ng-if="stepTimeField.showAdvancedOptions"
+  <!-- Index pattern id input -->
+  <div class="euiSpacer euiSpacer--m"></div>
+  <div ng-if="stepTimeField.showAdvancedOptions">
+    <label
+      class="euiLabel"
+      for="indexPatternCreationId"
     >
-      <label
-        class="kuiLabel kuiVerticalRhythmSmall"
-        for="indexPatternCreationId"
-      >
-        Custom index pattern ID
-      </label>
-
-      <div class="kuiVerticalRhythm kuiVerticalRhythmSmall">
-        <input
-          class="kuiTextInput kuiTextInput--large"
-          id="indexPatternCreationId"
-          data-test-subj="createIndexPatternIdInput"
-          ng-model="stepTimeField.indexPatternId"
-          ng-model-options="{ updateOn: 'default blur', debounce: {'default': 500, 'blur': 0} }"
-          validate-index-pattern
-          validate-index-pattern-allow-wildcard
-          name="id"
-          type="text"
-        >
-      </div>
+      Custom index pattern ID
+    </label>
 
-      <p
-        class="kuiText kuiSubduedText kuiVerticalRhythm"
-      >
-        Kibana will provide a unique identifier for each index pattern.
-        If you do not want to use this unique ID, enter a custom one.
-      </p>
-    </div>
+    <div class="euiSpacer euiSpacer--xs"></div>
+
+    <input
+      class="euiFieldText euiFieldText--large"
+      id="indexPatternCreationId"
+      data-test-subj="createIndexPatternIdInput"
+      ng-model="stepTimeField.indexPatternId"
+      ng-model-options="{ updateOn: 'default blur', debounce: {'default': 500, 'blur': 0} }"
+      validate-index-pattern
+      validate-index-pattern-allow-wildcard
+      name="id"
+      type="text"
+    >
+
+    <div class="euiSpacer euiSpacer--xs"></div>
+
+    <p class="euiTextColor euiTextColor--subdued">
+      Kibana will provide a unique identifier for each index pattern.
+      If you do not want to use this unique ID, enter a custom one.
+    </p>
+  </div>
 
-    <!-- Actions -->
-    <div class="kuiBar kuiVerticalRhythm">
-      <div class="kuiBarSection">
-        <div class="kuiButtonGroup">
-          <button
-            class="kuiButton kuiButton--secondary kuiButton--iconText"
-            ng-click="stepTimeField.goToPreviousStep()"
-          >
-            <span class="kuiButton__inner">
-              <span aria-hidden="true" class="kuiButton__icon kuiIcon fa-chevron-left"></span>
-              <span>Back</span>
-            </span>
-          </button>
-
-          <button
-            data-test-subj="createIndexPatternCreateButton"
-            ng-disabled="!stepTimeField.canCreateIndexPattern()"
-            class="kuiButton kuiButton--primary"
-            type="submit"
-          >
+  <div class="euiSpacer euiSpacer--m"></div>
+
+  <!-- Actions -->
+  <div class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--justifyContentSpaceAround euiFlexGroup--responsive">
```

Comment:
We can change lines 160-194 to:

```html
  <!-- Actions -->
  <div class="euiFlexGroup euiFlexGroup--gutterSmall euiFlexGroup--justifyContentFlexEnd">
    <div class="euiFlexItem euiFlexItem--flexGrowZero">
      <button
        class="euiButtonEmpty euiButtonEmpty--primary kuiButton--iconText"
        ng-click="stepTimeField.goToPreviousStep()"
      >
        <span class="euiButtonEmpty__content">
          <svg aria-hidden="true" class="euiIcon euiIcon--medium" xmlns="http://www.w3.org/2000/svg"
            xlink="http://www.w3.org/1999/xlink" width="16" height="16" viewBox="0 0 16 16">
            <defs>
              <path id="arrow_left-a" d="M13.069 5.157L8.384 9.768a.546.546 0 0 1-.768 0L2.93 5.158a.552.552 0 0 0-.771 0 .53.53 0 0 0 0 .759l4.684 4.61c.641.631 1.672.63 2.312 0l4.684-4.61a.53.53 0 0 0 0-.76.552.552 0 0 0-.771 0z"
              />
            </defs>
            <use fill-rule="nonzero" transform="rotate(90 8 8)" href="#arrow_left-a"
            />
          </svg>
          <span>Back</span>
        </span>
      </button>
    </div>
    <div class="euiFlexItem euiFlexItem--flexGrowZero">
      <button
        data-test-subj="createIndexPatternCreateButton"
        ng-disabled="!stepTimeField.canCreateIndexPattern()"
        class="euiButton euiButton--primary"
        type="submit"
      >
        <span class="euiButton__content">
          Create index pattern
        </span>
      </button>
    </div>
  </div>
```

This will render like this:

![image](https://user-images.githubusercontent.com/1238659/34229052-c1b16ca6-e587-11e7-885d-8b7242750f59.png)